src/desktop/apps/consign/components/create_account/index.tsx

```diff
@@ -20,6 +20,8 @@ export class CreateAccount extends React.Component<CreateAccountProps> {
       {
         copy: this.props.title,
         contextModule: "consignments",
+        destination: "/consign/submission",
```

Comment:
We only need `redirectTo`

Reply:
Thank you for the context! I'll make this change.